client/artifact.py

```diff
@@ -104,22 +112,34 @@ def __init__(self, repo, extension='.tar.gz'):
         self.pkgurl = pkg['url']
         self.pkgchksum = pkg['digests']['sha256']
 
-
 class MavenMetadata(object):
     """Retrieve the metadata for a Maven repository.
 
     Metadata is an XML (ElementTree) object.
     """
 
     __BASEURL = 'https://repo1.maven.org/maven2/{}'
+    __FRIENDLYURL = 'https://mvnrepository.com/artifact/{}'
     __METAFILE = 'maven-metadata.xml'
+    __TITLE = 'Maven Central'
+
+    def _path_to_friendly_url(self, path):
+        # Convert a.b.c.d to a.b.c/d
+        # E.g.,
+        #  com/nuodb/hibernate/nuodb-hibernate =>
+        #  com.nuodb.hibernate/nuodb-hibernate
```

Comment:
Something is odd about this comment: the statement on line 127 doesn't match the example at 129/130...?

Reply:
You are right, line 127 is wrong. Thank you for noticing. Will fix.